track.go

```diff
@@ -222,38 +223,43 @@ func (track *baseTrack) bind(ctx webrtc.TrackLocalContext, specializedTrack Trac
 	keyFrameController, ok := encodedReader.Controller().(codec.KeyFrameController)
 	if ok {
 		stopRead = make(chan struct{})
-		go func() {
-			reader := ctx.RTCPReader()
-			for {
-				select {
-				case <-stopRead:
-					return
-				default:
-				}
+		go track.rtcpReadLoop(ctx.RTCPReader(), keyFrameController, stopRead)
+	}
 
-				var readerBuffer []byte
-				_, _, err := reader.Read(readerBuffer, interceptor.Attributes{})
-				if err != nil {
-					track.onError(err)
-					return
-				}
+	return selectedCodec, nil
+}
+
+func (track *baseTrack) rtcpReadLoop(reader interceptor.RTCPReader, keyFrameController codec.KeyFrameController, stopRead chan struct{}) {
```

Comment:
Extracting the RTCP handler loop allow me to more easily unit test this part of the code.

The easiest way is to mock the RTCPReader since it is a mono function interface.